releases/2023.1/2023.1.md

```diff
@@ -0,0 +1,175 @@
+
+### 6. General UI fixes and enhancements 
+
+On version `2023.1`, the following UI components were fixed:
+
+- `ui` no logger supports `socket.io`
+- `ui` topology map now displays `metadata.node_name` as a default value when it's defined
+- `mef_eline` now supports `primary_constraints` and `secondary_constraints`, including correctly clearing these components
+
+<!---
+TODO ask Rogerio if he wants to include anything else or screenshots
+-->
+
+### 7. General core enhancements and endpoints OpenAPI validations
+
+Kytos-ng core `GenericEntity` has been augmented to expose `status` and `status_reason`, which as a result is exposed on `topology` too. `status` represents the entity status, and `status_reason` is a set of string values that gives further information about why an entity isn't UP, which also minimizes overuse of `metadata` key value pairs in certain cases.
+
+Kytos-ng core now exposes `@validate_openapi` decorator and an `async` function `avalidate_openapi_request(spec, request)` to validate API endpoints based an OpenAPI specification. On version `2023.1`, the following NApps are now validating their endpoints based on OpenAPI:
+
+- `sdntrace_cp`
+- `pathfinder`
+- `of_multi_table`
+- `mef_eline` (had already been using OpenAPI validations, but now it has included validations on additional endpoints)
+
+### 8. Specify requirements for `mef_eline` VLAN range feature and consolidated Kytos-ng documentation
+
+On version `2023.1`, Kytos-ng team has initially [specified requirements for supporting a range of VLANs](https://github.com/kytos-ng/kytos/issues/318) on `mef_eline` EVCs. Also, Kytos-ng team has specified [requirements to consolidate Kytos-ng documentation](https://github.com/kytos-ng/kytos/discussions/342), which will simplify and facilitate to onboard new users and contributors. Both of these specifications will be prioritized and implemented in a future release.
+
+### 9. Upgrade and simplify core development dependencies
+
+Kytos-ng now ships these extra runtime dependencies:
+
+- [`httpx`](https://github.com/encode/httpx)
+- [`openapi-core`](https://github.com/python-openapi/openapi-core)
+
+Also, Kytos-ng now provides the following development dependencies, that can be installed with `pip3.9 -e git+https://github.com/kytos-ng/kytos.git#egg=kytos[dev]`, allowing NApps to reuse these common development dependencies, which simplifies development requirements updates across all `kytos-ng` repositories:
+
+```
+pip-tools >= 2.0
+pytest==7.2.1
+pytest-cov==4.0.0
+pytest-asyncio==0.20.3
+black==23.3.0
+isort==5.12.0
+pylint==2.15.0
+pycodestyle==2.10.0
+yala==3.2.0
+tox==3.28.0
+virtualenv==20.21.0
+typing-extensions==4.5.0
+```
+
+## Database data model changes
```

Comment:
I ended moving DB data model changes as mentioned on yesterday's meeting, especially since this is crucial for production deployments, just so network operators can easily find out if any DB scripts are needed or not when using this version.